src/main/java/com/conveyal/gtfs/model/Stop.java

```diff
@@ -125,4 +198,185 @@ public Iterator<Stop> iterator() {
             return feed.stops.values().iterator();
         }   	
     }
+
+    /**
+     * Merge stop areas into stops when loading from file.
+     */
+    public static void mergeStopAreasIntoStops(Map<String, Stop> stops, Map<String, StopArea> stopAreas) {
+        Map<String, Set<String>> stopAreasByStopId = new HashMap<>();
+
+        stopAreas.values().forEach(stopArea ->
+            stopAreasByStopId
+                .computeIfAbsent(stopArea.stop_id, id -> new HashSet<>())
+                .add(stopArea.area_id)
+        );
+
+        stops.values().forEach(stop -> stop.stop_area_ids = getStopAreaIds(stopAreasByStopId, stop.stop_id));
+    }
+
+    /**
+     * Merge stop areas into stops when loading into DB.
+     */
+    public static CsvReader mergeStopAreasIntoStops(
```

Comment:
Rename to `getCsvReaderForStopsWithStopAreas` or similar, to make clear it is not an overload of the method above.